_posts/2022-03-25-emergent-symbols.md

```diff
@@ -0,0 +1,708 @@
+---
+layout: post
+title: Symbolic Binding in Neural Networks through Factorized Memory Systems
+tags: [symbolic, memory, neural, binding]
+authors: Ameya Daigavane, Ansh Khurana, Shweta Bhardwaj, Gaurav Aggarwal
+---
+
+Our blog post describes the paper
+[Emergent Symbols through Binding in External Memory](https://openreview.net/forum?id=LSFCEb3GYU7)
+that was published at ICLR 2021.
+
+## Introduction
+
+Human intelligence is often described as 'symbolic'
+because we are able to take complex sensory inputs and map
+them to abstract concepts, represented by logical rules and symbols.
+
+To illustrate this, consider the following scene with four objects
+of differing shape, size and color:
+
+{% include 2021-12-01-emergent-symbols/clevr.html %} 
```

Comment:
Could you update all of these paths to `2022-03-25-emergent-symbols/`? (throughout the whole document)

Reply:
Done!